src/os/bluestore/BlueStore.h

```diff
@@ -2224,7 +2225,8 @@ class BlueStore : public ObjectStore,
 
   uint64_t min_alloc_size; ///< minimum allocation unit (power of 2)
   ///< bits for min_alloc_size
```

Comment:
///< refers to text preceding the comment. If the comment refers to the next line, it should just be /// (without the '<')